src/helics/common/JsonProcessingFunctions.cpp

```diff
@@ -7,6 +7,7 @@ SPDX-License-Identifier: BSD-3-Clause
 
 #include "JsonProcessingFunctions.hpp"
 #include <fstream>
+#include <sstream>
```

Comment:
IF this is needed how was it working before?

Reply:
I'd guess it was getting included somewhere by json/json.h, but I don't know. It might be a problem when using the jsoncpp target in a Debug build.

Reply:
It's possible this is just a bandaid, and that the cause of the problem is in the json library -- will need to try a few more things and see; fixing it might be something for this weekend.

Reply:
Ahh,  this came up when you got rid of the other header?  I see now.